manifests/g/GitHub/cli/2.63.0/GitHub.cli.installer.yaml

```diff
@@ -14,22 +14,22 @@ InstallModes:
 UpgradeBehavior: install
 Commands:
 - gh
+ReleaseDate: 2024-11-27
 ElevationRequirement: elevatesSelf
 Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/cli/cli/releases/download/v2.63.0/gh_2.63.0_windows_amd64.msi
   InstallerSha256: E45595E9B416EA0933AA1FAAEDCBEA14C81D5512DA342C0219A4541CED76DCF7
   ProductCode: '{FAAF4306-25F7-422D-9758-CDFD6E6FC729}'
   AppsAndFeaturesEntries:
-  - ProductCode: '{C08A19D6-620B-415D-B4FA-0224B3D6F78E}'
+  - ProductCode: '{FAAF4306-25F7-422D-9758-CDFD6E6FC729}'
     UpgradeCode: '{8CFB9531-B959-4E1B-AA2E-4AF0FFCC4AF4}'

```

Comment:
```suggestion
  - UpgradeCode: '{8CFB9531-B959-4E1B-AA2E-4AF0FFCC4AF4}'
```

Reply:
I see, so there is no functional reason to include `ProductCode` twice. As long as it is at the root node it is fine? [I include it twice in manifests because the documentation states it should be when ARP node is present.](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.6.0/installer.md#:~:text=the%20ProductCode%20should%20be%20placed%20both%20within%20the%20installer%20and%20the%20AppsAndFeaturesEntries) I have wondered if it was necessary before.

Reply:
I believe that recommendation came from a time (probably WinGet 1.3) when CLI wouldn't match against the root value properly so the community decided to have it both at ARP and Installer level. But that should be not be the case anymore. In general, avoid ARP fields when there isn't a whole lot need for them (i.e., Package is having matching issues)

Reply:
Ahh I see. Thank you very much, @mdanish-kh. I will update this PR sometime today, and take note of your feedback for the future as well.